### PR TITLE
IN1.49 to Patient Ident; limit RelPers creation

### PR DESCRIPTION
**Pull request by cragun47**
Signed-off-by: Brian Cragun <cragun@us.ibm.com>

RelatedPerson should only be created when the subscribed is a relative. (Not an employer).

Patient.identifier from IN1.49.

### CODE REVIEW

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java [hunk 1]
```diff
@@ -99,15 +99,16 @@ void testBasicInsuranceCoverageFields() throws IOException {
...
                 // IN1.36 also to subscriberId
-                // IN1.46 to Identifier XV 3
-                // IN1.47 through IN1.53 NOT REFERENCED
-                + "|MEMBER36||||||||||Value46|||||||\n"
-                // IN2.6 is purposely empty so will not create an MC identifier
-                // IN2.8 is purposely empty so will not create an MA identifier
+                // IN1.46 to Coverage.identifier XV 3
+                // IN1.49 to PatientCoverage.identifier
+                // IN1.53 through IN1.53 NOT REFERENCED
+                + "|MEMBER36||||||||||Value46||PatientId48|PatientId49.1^^^System49.4^TypeCode49.5|PatientId50|||\n"
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:107]: Looks like also added IN1.50 not sure why

**Review by LisaWellman — APPROVED** [review]
Looks good!

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java [hunk 2]
```diff
@@ -99,15 +99,16 @@ void testBasicInsuranceCoverageFields() throws IOException {
...
+                // IN1.36 to Coverage.identifier MB and Coverage.identifier SN
                 // IN1.36 also to subscriberId
-                // IN1.46 to Identifier XV 3
-                // IN1.47 through IN1.53 NOT REFERENCED
-                + "|MEMBER36||||||||||Value46|||||||\n"
-                // IN2.6 is purposely empty so will not create an MC identifier
-                // IN2.8 is purposely empty so will not create an MA identifier
+                // IN1.46 to Coverage.identifier XV 3
+                // IN1.49 to PatientCoverage.identifier
+                // IN1.53 through IN1.53 NOT REFERENCED
```
[the remark this answers is not in the document]
Comment (cragun47) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:106]: DOne

[the remark this answers is not in the document]
Comment (cragun47) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:106]: Yeah, it used to be a larger range, but it got whittled down.

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java (hunk 1 above)
Comment (cragun47) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:107]: Oh darn.  Yes, when I have troubles hitting the field number of these long-gaps, I sometimes put information in the nearby fields to see if its my count or the yaml that's failing.  I forgot to remove this one.  Good catch.  Remove IN1.48 as well.

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java [hunk 3]
```diff
@@ -99,15 +99,16 @@ void testBasicInsuranceCoverageFields() throws IOException {
...
                 + "|20201231145045|20211231145045|||||||||5|||||||||||||COMPANYPLANCODE35"
-                // IN1.36 to Identifier MB and Identifier SN
+                // IN1.36 to Coverage.identifier MB and Coverage.identifier SN
                 // IN1.36 also to subscriberId
-                // IN1.46 to Identifier XV 3
-                // IN1.47 through IN1.53 NOT REFERENCED
-                + "|MEMBER36||||||||||Value46|||||||\n"
-                // IN2.6 is purposely empty so will not create an MC identifier
-                // IN2.8 is purposely empty so will not create an MA identifier
+                // IN1.46 to Coverage.identifier XV 3
```
[the remark this answers is not in the document]
Comment (cragun47) [reply src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:104]: There are 3 XV ids created.  I'll clarify. with "third XV Coverage.identifier."

**Pull request merged**